.github/workflows/dev-deploy.yml

```diff
@@ -7,7 +7,7 @@ on:
 
 jobs:
   build:
-    uses: axonivy-market/github-workflows/.github/workflows/dev.yml@v4
+    uses: axonivy-market/github-workflows/.github/workflows/dev.yml@v5
     with:
       ivyVersion: 11
       ivyPluginVersion: 11.3.0
```

Comment:
I think in the dev.yml the version parameters should be removed too; we should go with the default and therefore test 13.1

Reply:
Ah, we use this script to deploy our current code to a server, not a true dev build :sweat_smile:
However, I am aware that our dev-build does not follow the legacy idea so I updated it in https://github.com/axonivy-market/process-analyser/pull/69/commits/1841a8a0caec83a76e016d85895f1a848f4cd255 
Thanks for reviewing